cadccutout/cadccutout/transform.py

```diff
@@ -126,19 +142,21 @@ def __init__(self, header):
                     self.spatial_1 = i + 1
                 else:
                     self.spatial_2 = i + 1
-                    logger.info('Spatial naxis{} naxis{}'.format(
+                    logging.info('Spatial naxis{} naxis{}'.format(
```

Comment:
In package use the logger for the package instead of the default, root one. Comment applicable to entire package.